styles/namemc/catppuccin.user.css

```diff
@@ -0,0 +1,582 @@
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    --bs-body-color: @text;
+    --bs-body-color-rgb: @text;
+    --bs-body-bg: @base;
+    --bs-body-bg-rgb: @base;
+    --bs-emphasis-color: @text;
+    --bs-emphasis-color-rgb: @text;
+    --bs-blue: @blue;
+    --bs-indigo: @mauve;
+    --bs-purple: @mauve;
+    --bs-pink: @pink;
+    --bs-red: @red;
+    --bs-orange: @peach;
+    --bs-yellow: @yellow;
+    --bs-green: @green;
+    --bs-teal: @teal;
+    --bs-cyan: @sapphire;
+    --bs-secondary-color: @subtext0;
+    --bs-secondary-color-rgb: @subtext0;
+    --bs-tertiary-color: @subtext1;
+    --bs-tertiary-color-rgb: @subtext1;
+    --bs-tertiary-bg: @base;
+    --bs-tertiary-bg-rgb: @base;
+    --bs-secondary-text-emphasis: @subtext0;
+    --bs-success-text-emphasis: @accent-color;
+    --bs-primary: @accent-color;
+    --bs-primary-rgb: @accent-color;
+    --bs-secondary: @subtext0;
+    --bs-secondary-rgb: @subtext0;
+    --bs-success: @green;
+    --bs-success-rgb: @green;
+    --bs-info: @sky;
+    --bs-info-rgb: @sky;
+    --bs-warning: @yellow;
+    --bs-warning-rgb: @yellow;
+    --bs-danger: @red;
+    --bs-danger-rgb: @red;
+    --bs-light: @base;
+    --bs-light-rgb: @base;
+    --bs-dark: @text;
+    --bs-dark-rgb: @text;
+    --bs-dark-text-emphasis: @text;
+    --bs-primary-bg-subtle: @crust;
+    --bs-secondary-bg-subtle: @mantle;
+    --bs-success-bg-subtle: @crust;
+    --bs-warning-bg-subtle: @surface0;
+    --bs-danger-bg-subtle: @base;
+    --bs-secondary-border-subtle: @surface1;
+    --bs-success-border-subtle: @accent-color;
+    --bs-info-border-subtle: @sky;
+    --bs-warning-border-subtle: @peach;
+    --bs-danger-border-subtle: @red;
+    --bs-link-color: @accent-color;
+    --bs-link-color-rgb: @accent-color;
+    --bs-code-color: @accent-color;
+    --bs-highlight-bg: @base;
+    --bs-form-valid-color: @green;
+    --bs-focus-ring-color: @accent-color;
+    --bs-form-valid-border-color: @green;
+    --bs-form-invalid-color: @red;
+    --bs-form-invalid-border-color: @red;
+
+    #bsButton(@col, @bgCol) {
+      --bs-btn-color: @col;
+      --bs-btn-hover-color: @col;
+      --bs-btn-active-color: @col;
+      --bs-btn-disabled-color: @col;
+
+      --bs-btn-bg: @bgCol;
+      --bs-btn-hover-bg: @bgCol;
+      --bs-btn-active-bg: @bgCol;
```

Comment:
The hover/active color should be different from the first color. `darken(@accent-color, 5%)` would work.